src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -469,7 +469,7 @@ private TileCraftingTile getCore()
 		{
 			return null;
 		}
-		return (TileCraftingTile) this.machineSrc.via;
+		return (TileCraftingTile) this.machineSrc.machine().get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3655)